### ISSUE TITLE
Grafana dashboards are not sent through the proxy

### ISSUE THREAD

**Issue by simskij** [canonical/cos-proxy-operator#24]
### Enhancement Proposal

A lot of the LMA compatible charms of today do send dashboards over to Grafana from the providing side. COS Proxy does not currently support this. For an example; have a look at the [telegraf charm on Launchpad](https://git.launchpad.net/charm-telegraf/tree/src/templates/dashboards/grafana).

This is not the same as #7, which is a method that does not need to be supported anymore, according to @aieri.

**Comment by peppepetra** [comment]
I have tested this and I can import existing dashboards via cos-proxy from channel edge rev. 9

I tested it with prometheus-openstack-dashboard that implements the legacy "dashboards" relation.

To make it working I have to relate cos-proxy with the offer from the cos-lite model grafana-dashboads:

```
juju add-relation cos-proxy:downstream-grafana-dashboard grafana-dashboards:grafana-dashboard 
```

and then relate prometheus-openstack-exporter with cos-proxy with:

```
juju add-relation prometheus-openstack-exporter:dashboards cos-proxy:dashboards
```

after this I have the prometheus-openstack-exporter dashboards correctly imported in grafana-k8s

**Comment by rbarry82** [comment]
Closed as part of #31
